src/ResourceManager/Network/ChangeLog.md

```diff
@@ -32,6 +32,12 @@
     - Updated Add-AzureRMLoadBalancerRuleConfig: Added optional parameter DisableOutboundSNAT
     - Updated Set-AzureRMLoadBalancerRuleConfig: Added optional parameter DisableOutboundSNAT
 
+## Version 4.3.2
+* New-AzureRmVirtualNetworkGateway: Added IkeV2 P2S support. New optional parameter added
```

Comment:
@henry416 please remove the `## Version 4.3.2` header and move the contents you are adding to the `## Current Release` section